### PR TITLE
Migrate under parent pom for odre server

### PR DESCRIPTION
**Pull request by etiennehomer**
Signed-off-by: Etienne <etiennehomer@gmai.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -145,82 +141,13 @@
                 <plugin>
                     <groupId>com.google.cloud.tools</groupId>
                     <artifactId>jib-maven-plugin</artifactId>
```
Comment (jonenst) [inline pom.xml:143]: should be removed

File: pom.xml [hunk 2]
```diff
@@ -232,36 +159,6 @@
             <activation>
```
Comment (jonenst) [inline pom.xml:159]: profile should be removed

File: src/main/java/org/gridsuite/odre/server/utils/DistanceCalculator.java [hunk 3]
```diff
@@ -12,6 +12,7 @@
 /**
  * @author Chamseddine Benhamed <chamseddine.benhamed at rte-france.com>
  */
+@SuppressWarnings("checkstyle:HideUtilityClassConstructor")
```
Comment (jonenst) [inline src/main/java/org/gridsuite/odre/server/utils/DistanceCalculator.java:15]: should fix the warning instead of suppressing it in this case

Comment (ChamseddineBhd) [reply src/main/java/org/gridsuite/odre/server/utils/DistanceCalculator.java:15]: same remark

**Pull request merged**